app/app.py

```diff
@@ -109,10 +109,26 @@ def find_recipe_in_url_window_algo_based():
 
     ingred_paragraph, instr_paragraph = extract(ingredients, instructions, url)
 
+    # TODO - use remove_unwanted_patterns - if we want to support english also
+    # ingred_paragraph = remove_unwanted_patterns(ingred_paragraph)
+    # instr_paragraph = remove_unwanted_patterns(instr_paragraph)
+
     return {'ingredients': ingred_paragraph,
             'instructions': instr_paragraph}
 
 
+
```

Comment:
app.py is already dirty with logic which is not related to the end-point.
lets move this method to where it belong to - i think its textExtractor